nginx/ng/src.sls

```diff
@@ -29,7 +29,7 @@ nginx_download:
 
 nginx_configure:
   cmd.run:
-    - name: ./configure --prefix=/etc/nginx --sbin-path=/usr/sbin/nginx --conf-path=/etc/nginx/nginx.conf {{ nginx.source.opts | join(' ') }}
+    - name: ./configure --prefix=/etc/nginx --sbin-path=/usr/sbin/nginx --conf-path={{ nginx.lookup.conf_file }} {{ nginx.source.opts | join(' ') }}
```

Comment:
Could you please add `/etc/nginx/nginx.conf` back as the default? Then this will continue to be backwards compatible.